src/index.html

```diff
@@ -361,29 +361,132 @@ <h3>Lunch & Coffee</h3>
             <p>Enjoy the best coffee, snacks and delicious lunch.</p>
           </div>
         </div>
-      </section>
+      </section> -->
 
-      <section id="sponsor_offer" class="sponsor-offer">
-        <div class="sponsor-offer__content">
-          <div class="sponsor-offer__main-info">
-            <h2 class="sponsor-offer__heading"><span aria-hidden="true">#</span>sponsors</h2>
+      <section id="sponsor_list" class="sponsor-list">
+        <div class="sponsor-list__content">
+          <div class="sponsor-list__main-info">
+            <h2 class="sponsor-list__heading"><span aria-hidden="true">#</span>sponsors</h2>
           </div>
-          <p class="sponsor-offer__text">
+          <p class="sponsor-list__text">
             Each year, our sponsors and partners help us keep tickets affordable, organize
             a&nbsp;great event and welcome you in their stands.
           </p>
 
-          <h3 class="sponsor-offer__subheading"><span>Platinum sponsor</span></h3>
-          <ul class="sponsor-offer__list">
-            <li class="sponsor-offer__list-item">
+          <h3 class="sponsor-list__subheading"><span>Platinum sponsor</span></h3>
+          <ul class="sponsor-list__list">
+            <li class="sponsor-list__list-item">
               <a href="https://tsh.io/" title="The Software House" target="_blank">
                 <img alt="The Software House" src="images/sponsors/platinum/tsh.svg" />
               </a>
             </li>
+            <li class="sponsor-list__list-item">
+              <a
+                href="#sponsor_offer"
+                class="sponsor-list__list-item--offer-link"
+                title="Become a sponsor"
```

Comment:
There can be only one main

Reply:
Is there already main sponsor? Should I remove this tile?